src/main/java/frc/robot/commands/groups/ShooterRevCommandgroup.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.commands.groups;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Robot;
+import frc.robot.commands.FeederSafetyReverseCommand;
+
+public class ShooterRevCommandgroup extends SequentialCommandGroup{
+
+    public ShooterRevCommandgroup() {
+        addCommands(
+            Robot.FeederSafetyReverse,
```

Comment:
The FeederSafetyReverse command can be replaced with ParallelDeadlineGroup (with a WaitCommand being the deadline) and a new StartEndCommand.

Reply:
ShooterRevCommandGroup is unused anyway, as it causes other issues- deleted file entirely